serverless/deploy/github/witness_pr/entrypoint.sh

```diff
@@ -11,26 +11,28 @@ set -e
 function main {
     cd ${GITHUB_WORKSPACE}
 
-    LOG_DIR="$(readlink -f -n ${INPUT_PR_REPO_ROOT}/${INPUT_LOG_DIR})"
-    LOG_CP="$(readlink -f -n ${INPUT_PR_REPO_ROOT}/${INPUT_LOG_DIR}/checkpoint)"
-    MODIFIED_CP="$(readlink -f -n ${INPUT_PR_REPO_ROOT}/${INPUT_LOG_DIR}/checkpoint.witnessed)"
+    LOG_DIR="$(readlink -f -n ${INPUT_LOG_DIR})"
 
-    cd ${INPUT_PR_REPO_ROOT}
     # Now grab a list of all the modified/added/removed files in the PR
     FILES=$(git diff origin/master HEAD --name-only)
 
     local has_non_log_files=0
     local has_witnessed_checkpoint=0
     local has_log_non_witness_files=0
+    declare -a witness_checkpoints
 
     while IFS= read -r f; do
             f=$(readlink -f -n ${f})
-            if [[ ${f} = ${LOG_CP} ]]; then
-                echo "::error: file=${f} Modifications to checkpoint not allowed, please use checkpoint.witnessed"
-                has_witnessed_checkpoint=1
-            elif [[ ${f} = ${MODIFIED_CP} ]]; then
-                echo "::debug:Found checkpoint.witnessed"
+            if [[ ${f} = ${LOG_DIR}/checkpoint ]]; then
+                echo "::error: file=${f} Modifications to checkpoint not allowed, please add files to the witnessed/ directory."
```

Comment:
This says to add to the `witnessed` directory, but the check below is for `witness/*` ?

Whichever is the correct directory should be stated in the docs at `serverless/deploy/github/witness_pr/README.md`.

Reply:
OK, from looking at other PRs it seems that this doc is wrong but the code is right below. Approving this PR with the expectation this docs and the README will be clarified before merging.

Reply:
good spot, fixed - I'm going to blame that on muscle memory :)